drivers/dai/intel/ssp/ssp.h

```diff
@@ -7,6 +7,24 @@
 #ifndef __INTEL_DAI_DRIVER_SSP_H__
 #define __INTEL_DAI_DRIVER_SSP_H__
 
+#define SSP_IP_VER_1_0 0x10000 /* cAVS */
+#define SSP_IP_VER_1_5 0x10500 /* MTL */
+#define SSP_IP_VER_2_0 0x20000 /* LNL */
+#define SSP_IP_VER_3_0 0x30000 /* ACE30 */
```

Comment:
To be consistent, I would just replace the names in the comments: MTL with ACE15 and LNL with ACE20.